docs: url updates (#1563)
spectrocloud/librarium: docs/docs-content/clusters/public-cloud/azure/aks.md

<br />

<video title="aks-cluster-creation" src="/videos/clusters/public-cloud/cloud-accounts/azure.mp4"></video>

The following steps need to be performed to provision a new cluster:
<br />

> 1. If you already have a profile to use, go to the **Cluster** > **Add a New Cluster** > **Deploy New Cluster** and select an Azure cloud. If you do not have a profile to use, reference the [Creating a Cluster Profile](../../../cluster-profiles/task-define-profile.md) page for steps on how to create one.

Reviewer: ```suggestion
1. If you already have a profile to use, go to **Cluster** > **Add a New Cluster** > **Deploy New Cluster** and select an Azure cloud. If you do not have a profile to use, review the [Creating a Cluster Profile](../../../cluster-profiles/task-define-profile.md) page for steps to create one.

```
